### DISCUSSION TITLE
How Salmon deals with poly(N) in a genome/transcripts?

### DISCUSSION THREAD

**Discussion by ignadb** [COMBINE-lab/salmon#686] [Q&A (bulk specific)]
Hi Salmon developers and folks,

I have a draft genome (and its RNA-Seq data) containing stretches of Ns. I wonder how Salmon deals with these poly(N) during mapping and whether it is OK to use the draft genome for constructing a decoy?

Thanks in advance and stay safe from covid-19!

Best regards

**Comment by mdshw5** [comment]
I believe that salmon incorporates the N's into the transcript length, and during mapping will not seed MEMs in regions of "N". In practice this should mean that unique sequences flanking stretches of "N" will be used for mapping reads, and you can use a draft genome for constructing a decoy.
